demo/helpers.ts

```diff
@@ -5,7 +5,7 @@ import type { Asset } from "../src/elements/helpers/defaultTransform";
 import type { SetMedia } from "../src/elements/helpers/types/Media";
 
 type GridAsset = {
-  mimeType: string;
+  mimeType: "image/jpeg" | "image/png" | "image/svg+xml";
```

Comment:
are we positive these are the only values? (appreciate these will be gone at runtime, so can't imagine it breaking anything, just want to be sure we're not giving anyone false sense of security down the line)

Reply:
I have no idea to be honest! Who'd be the person to ask?

I just copied from the demo implementation in DCR (linked to in the ticket):

https://github.com/guardian/dotcom-rendering/pull/6706/files#diff-c8083dc35c9849f252a62cef0888fc89dbe373563decd9d4776692b4c47cec12R12